### ISSUE TITLE
Add unit tests where appropriate

### ISSUE THREAD

**Issue by mattwatson** [DaveEmmerson/SpaceWar2#13]

**Comment by DaveEmmerson** [comment]
Do you use nunit? I have it loosely roped to my vs express as it can't be installed properly.

I'm happy to use another tool if it's free!

**Comment by mattwatson** [comment]
Yeah I tend to use nunit - its really good. It can run standalone, but I
tend to run it through resharper's test runner

**Comment by DaveEmmerson** [comment]
Separate project for tests?

(using the assembly friend thingy so they can access the main project)

I'm not sure why, but I always have a separate project for it... something I read somewhere sometime.

**Comment by mattwatson** [comment]
Yes separate project. You can use the same namespace to access internal
stuff I think. But if you are only testing the public methods then u don't
need to do that.

**Comment by DaveEmmerson** [comment]
This is done. Well, the framework is there. We just need loads of tests ;)

I suggest we close this issue and open up specific ones where we need to write tests for known bugs etc?
